src/utils/quote_report_utils.ts

```diff
@@ -18,7 +18,18 @@ type QuoteReportLogOptions = QuoteReportForTakerTxn | QuoteReportForMetaTxn;
 
 export const quoteReportUtils = {
```

Comment:
now that we are logging indicative and firm quotes -- would it be easy to log whether or not this was firm or indicative in the `logQuoteReport` fn ? i think this would be useful, but we also would have to update the metabase consumer to support this

Reply:
I might missing something but nothing should have changed regarding when we log reports.

We are only logging from the `/quote` endpoints if an api key is supplied. Is that not correct?

Reply:
you are right -- my bad, i misread what was happening -- got a bit mixed up with us generating quote repots but not logging them.  thanks for the clarification.